Yeti-experience/draft-song-yeti-testbed-experience.txt

```diff
@@ -277,134 +319,118 @@ Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
+  `----+----+----'    `---+---'    `---+---'    `---+---'
+       |    |       ,-----'    ,-------'            `----.
+       |    |       |          |                         |
+       ^    ^       |          |                         |     Yeti-Root
+       |    |   ,---V---.  ,---V---.                 ,---V---.
+       |    `---+ Yeti  |  | Yeti  |  . . . . . . .  | Yeti  |
+       |        | Root  |  | Root  |                 | Root  |
+       |        `---+---'  `---+---'                 `---+---'
+       |            |          |                         |
+       |            |          |                         |     DNS Response
+       |          ,--V----------V-------------------------V--.
+       `----------+                Yeti Resolvers            |
+                  `-----------------------+------------------'
+                                          |
+                                          |                    DNS Response
+                  ,-----------------------V------------------.
+                  |              Yeti Stub Resolvers         |
+                  `------------------------------------------'
+
+
+                        Figure 2: Testbed Data Flow
+
+3.1.  Root Zone Retrieval
+
+   Since Yeti DNS servers cannot receive DNS NOTIFY [RFC1996] messages
+   from the Root Server System, a polling approach is used.  Each Yeti
+   Distribution Master (DM) requests the root zone SOA record from
+   XXX(which servers?)XXX and performs a zone transfer if the SOA.SERIAL
+   value is greater than that of the last retrieved zone.
+
+   The schedule on which the Root Server System is polled by each Yeti
+   DM is as follows:
+
+                  +-------------+-----------------------+
+                  | DM Operator | Time                  |
+                  +-------------+-----------------------+
+                  | BII         | UTC hour + 00 minutes |
+                  | WIDE        | UTC hour + 20 minutes |
+                  | TISF        | UTC hour + 40 minutes |
+                  +-------------+-----------------------+
+
+
+
+Song, et al.            Expires December 18, 2017               [Page 7]
 
 Internet-Draft      Yeti DNS Testbed: Interim Results          June 2017
 
 
-   While in principle this could be done by a single DM, Yeti uses a set
-   of three DMs to avoid any sense that the Yeti DNS Project is run by a
-   single organization.  Each of three DMs independently fetches the
-   root zone from IANA, signs it and publishes the latest zone data to
-   Yeti root servers.
+   Zone transfers by Yeti DMs are carried out using AXFR from XXX(which
+   servers?)XXX with no authentication.
 
-   In the same while, these DMs coordinate their work so that the
-   resulting Yeti root zone is always consistent.  There are two aspects
-   of coordination between three DMs: timing and information
-   synchronization.
+3.2.  Transformation of Root Zone to Yeti-Root Zone
```

Comment:
The routine of DM are documented in : https://github.com/BII-Lab/Yeti-Project/blob/master/doc/Yeti-DM-Setup.md